src/stable_browser.ts

```
@@ -345,6 +345,9 @@ class StableBrowser {
     return locatorReturn;
   }
   async _locateElmentByTextClimbCss(scope, text, climb, css, _params: Params) {
+    if (css && css.locator) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)